client/scss/components/_numbered_list.scss

```diff
@@ -53,6 +53,7 @@ $numbered-list-number-size-large: 96px;
 }
 
 .numbered-list__number {
+  font-family: $wellcome-bold, $wellcome-bold-subset, $wellcome-bold-base;
```

Comment:
Previously, `@include font('WB1')` was declared here, but then the font-size was being overridden on the line below. I didn't notice this override when adding the font classes to the markup.